core/src/main/java/com/bankbazaar/tripmanager/controller/RegisterController.java

```diff
@@ -0,0 +1,53 @@
+package com.bankbazaar.tripmanager.controller;
+
+
+import com.bankbazaar.tripmanager.manager.RegisterManager;
+import com.bankbazaar.tripmanager.model.Users;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+
+import static org.hibernate.tool.schema.SchemaToolingLogging.LOGGER;
```

Comment:
Use @Slf4j. Spring boot comes with logback logger

Reply:
@Sl4j used to logging